WordPress/Classes/ViewRelated/Suggestions/SuggestionsTableView.m

```diff
@@ -91,8 +91,13 @@ - (void)setupViewModel
     __weak __typeof(self) weakSelf = self;
     self.viewModel.searchResultUpdated = ^(id<SuggestionsListViewModelType> viewModel) {
         if (!weakSelf) return;
+        CGFloat sectionHeaderAndFooterHeight = viewModel.sections.count > 1 ? -1 : CGFLOAT_MIN;
```

Comment:
So there are some cases in UIKit where setting a height to `0` doesn't mean "no height", instead it means "default height value". 

For example, I'm using this code to remove some padding at the bottom of the table view:
```
// Removes a small padding at the bottom of the tableView
UIView *footerView = [[UIView alloc] initWithFrame:CGRectMake(0, 0, 0, CGFLOAT_MIN)];
footerView.backgroundColor = [UIColor clearColor];
[_tableView setTableFooterView:footerView];
```
If we replace `CGFLOAT_MIN` with `0`, the height of that view won't be really `0`. See the following screenshot:

![CleanShot 2022-07-28 at 20 07 28@2x](https://user-images.githubusercontent.com/9609223/181618981-f1133ae2-97c7-4096-849d-945c19de2115.png)

Here is another [example](https://stackoverflow.com/questions/19056428/how-to-hide-first-section-header-in-uitableview-grouped-style) where `0` also doesn't work as expected. So knowing this, `CGFLOAT_MIN` ( and its Swift version ) became like a habit.

That being said, apparently `0` does work as expected for `tableView.sectionHeaderHeight`, I think the reason behind that is because that property uses the value `-1` to indicate "default height" and it leaves `0` to indicate "no height".

I'll make the changes. Let me know if this makes sense.

Reply:
Thanks for the explanation @salimbraksa!